### ISSUE TITLE
Quasar project decimal comma input does not work on android 9

### ISSUE THREAD

**Issue by happymax-dbs** [dm4t2/vue-currency-input#190]
I use the last version 1.22.4 with locale set to "it" - this is my function method in computed block

```
currencyOptions () {
       return {
          locale: 'it',
          currency: null,
          allowNegative: false
       }

```
When running the application on the Android 9 phone or tablet, the decimal point (comma) does not work.

I have tested also your playground page and the only configuration that work is locale en-IN or en-US; any configuration with comma as decimal separator dont't work.

Can you please give me some advice to solve this issue.
Thanks a lot for you work and your help.

**Comment by dm4t2** [comment]
Hi, thanks for reporting. I can reproduce your issue on BrowserStack. It seems, that there's no `keypress` event triggered on Android when using the numeric keyboard. This is necessary to input decimal separators different from the current locale (see also #71, #126).

**Comment by dm4t2** [comment]
Fixed in [2.0.0-beta.3](https://github.com/dm4t2/vue-currency-input/releases/tag/2.0.0-beta.3)  🎉
